graphql/resolvers.go

```diff
@@ -2042,6 +2042,29 @@ func (r *queryResolver) BuildBaron(ctx context.Context, taskId string, exec int)
 	}, nil
 }
 
+func (r *mutationResolver) BbFileTicket(ctx context.Context, taskId string) (bool, error) {
+	taskNotFound, err := BbFileTicket(taskId)
```

Comment:
Not a big deal but I would rename one of these functions so that it doesn't look like this is recursive

Reply:
good point! thanks!